.github/workflows/system-tests.yml

```diff
@@ -0,0 +1,57 @@
+
+name: System Tests
+
+on:
+  push:
+    branches:
+      - "**"
+  # pull_request:
+  #   branches:
+  #     - "**"
```

Comment:
I'm more in favor of enabling the`pull_request` trigger on every branch because it is performed on the merge result of the pull request, which is really valuable to check the resulting merge works properly.

On the other hand, I think we could limit the `push` trigger just to `v1` commits.

I would also add the `tag` trigger.

Reply:
typo of myself, the push was here for testing before creating the PR